.github/workflows/publish.yml

```diff
@@ -0,0 +1,26 @@
+name: Cargo Publish
+
+on:
+  push:
+    tags:
+      - 'v**'
+
+permissions:
+  contents: read
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    environment: crates.io
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 #v4.2.2
+      - uses: actions-rs/toolchain@16499b5e05bf2e26879000db0c1d13f7e13fa3af #v1.0.7
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
```

Comment:
Why not take the time to use the actively maintain action for this new workflow? Actually, I suppose it would be more strange to deviate from the one used in the other workflow so never mind. We can update them both separately.

Reply:
Yeah I think we should get this one working then change them both in another PR.